src/adaXT/criteria/criteria.pyx

```diff
@@ -467,10 +467,10 @@ cdef class Squared_error(Criteria):
         square_err = cur_sum/obs_weight - mu*mu
         return square_err
 
-cdef class Linear_regression(Criteria):
+cdef class Partial_linear(Criteria):
 
     # Custom mean function, such that we don't have to loop through twice.
-    cdef (double, double) custom_mean(self, int[:] indices):
+    cdef (double, double) _custom_mean(self, int[:] indices):
```

Comment:
If we see them as only private functions I think the general convention is to use a two underscores (__) for private methods.

Reply:
changed everything to double dash now